static/scss/course_details/tiles.scss

```diff
@@ -0,0 +1,95 @@
+// sass-lint:disable mixins-before-declarations
+
+.tiles-block {
+  width: 100%;
+  padding: 32px 0;
+  background: $tiles-background url('#{$static-path}/images/tiles-bg.png') repeat-x 0 100%;
+  min-height: 200px;
+  @include media-breakpoint-up(lg) {
+    background-position: 0 100%;
+  }
+}
+
+
+.tiles-list {
+  letter-spacing: -.32em;
+  margin: 0 -8px;
+  padding: 0;
+  list-style: none;
+  text-align: center;
+
+  li {
+    letter-spacing: normal;
+    background: $tile-background;
+    border-radius: 5px;
+    margin: 8px;
+    padding: 32px 10px;
+    @include media-breakpoint-up(lg) {
+      display: inline-block;
+      vertical-align: top;
+      width: calc(20% - 16px);
+      min-height: 215px;
+      padding: 32px 10px 10px;
+    }
+  }
+
+  .title {
+    display: block;
+    font-size: 18px;
+    font-weight: bold;
+    color: $red;
+    text-align: center;
+    position: relative;
+    padding: 0 0 5px;
+    line-height: 22px;
+    text-transform: uppercase;
+    margin: 0 0 15px;
+
+    &:after {
+      position: absolute;
+      left: 50%;
+      top: 100%;
+      border-bottom: 1px solid $red;
+      width: 100px;
+      content: '';
+      margin: 0 0 0 -50px;
+    }
+  }
+
+  .text {
+    display: block;
+    font-size: 48px;
+    line-height: 50px;
+    color: $gray;
+    font-weight: 600;
+    text-align: center;
+
+    @include media-breakpoint-up(lg) {
+      font-size: 38px;
+      line-height: 50px;
+    }
+
+    @include media-breakpoint-up(xl) {
```

Comment:
I am seeing this breakpoint (`xl`) only for `.text`? do we need to handle this in other classes as well ?

Reply:
I'll take a look at different screen widths and see if anything looks like it needs fixing